app/javascript/controllers/ordered_form_controller.js

```diff
@@ -0,0 +1,72 @@
+import NestedForm from 'stimulus-rails-nested-form'
+
+export default class extends NestedForm {
```

Comment:
This is based on our existing stimulus controller for supporting ordering, but extends NestedForm instead of our own custom nested form controller implementation.